frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
   - Publication/preprint DOI: can be added later
```

Comment:
When Max wrote this section, @jahilton was the primary contributor and reviewers. Jason - any comments on [frontend/doc-site/032__Contribute and Publish Data.mdx](https://github.com/chanzuckerberg/single-cell-data-portal/pull/3180/files#diff-3aab6608e02aeee4406a95606f0a84d5f03989f2ababdb4fb87416f1eb5f8c7e) ?